### ISSUE TITLE
Source code license?

### ISSUE THREAD

**Issue by Kiv** [mtgred/netrunner#402]
It would be nice to know what license the code is available under - both for contributors to know what happens to the copyright of their contributions, and for anyone that might want to make a derivative work such as using part of the rules engine for another card game.

**Comment by nealterrell** [comment]
@mtgred ?

**Comment by mtgred** [comment]
This needs to be addressed indeed. I don't have a good knowledge about the differences the various open source licenses. I need to do some homework first.

**Comment by aaronthewhite** [comment]
@mtgred 
This is a good tool for narrowing down the licenses:
http://oss-watch.ac.uk/apps/licdiff/

A site that lists and explains the most popular licenses:
http://choosealicense.com/
They also have a list of licenses in a format that is easier to compare:
http://choosealicense.com/licenses

This site is good for explaining a license in plain english:
https://tldrlegal.com/

Ultimately, it just depends on what you want to allow.

**Comment by steveklabnik** [comment]
Came here to ask about this. Until you pick a license, this code is technically "all rights reserved". It also means that the contributions of other people are technically not under that license either...

> Ultimately, it just depends on what you want to allow.

Yup, this is it. @mtgred if you can explain a bit about what specifically you want with regards to sharing, I'd be happy to help explain some licenses.

IMHO, the biggest quesiton is "Are you okay with someone taking this code, using it in something closed source, and then distributing it to others?" Many are, many aren't.

**Comment by jhli** [comment]
If you want share back:
AGPL, ALGPL
http://stackoverflow.com/questions/2127246/difference-between-affero-gpl-and-gplv3
If you don't want that: MIT.

**Comment by zaroth** [comment]
As to add to the discussion, I'd like to note that just submitting code to open repository on Github subjects it to some [Terms of Service](https://help.github.com/articles/open-source-licensing/#what-happens-if-i-dont-choose-a-license), so forking and viewing is allowed.

**Comment by queueseven** [comment]
Finally
